SCons/Script/Main.py

```diff
@@ -492,6 +492,19 @@ def GetOption(name):
 def SetOption(name, value):
     return OptionsParser.values.set_option(name, value)
 
+
+def ValidateOptions(throw_exception=False):
+    """
+    If you call this after you set all your command line options with AddOption(),
```

Comment:
Docstring should (per style guides - but much of SCons doesn't do this) start with a single-line statement of purpose, then after a blank line, go on to the rest of, or the fuller, description.  Maybe also end with a `Raises:` section?  Not crucial, just makes the API docs look prettier :)